src/DependencyInjection/Configuration.php

```diff
@@ -37,6 +37,55 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->addDefaultsIfNotSet()
             ->children()
+                ->arrayNode('pdf_rendering')
+                ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('default_orientation')->defaultValue('Portrait')->end()
+                        ->arrayNode('available_orientations')
+                            ->scalarPrototype()->end()
+                            ->defaultValue([
+                                'Portrait',
+                                'Landscape',
+                            ])
+                        ->end()
+                        ->scalarNode('default_page_size')->defaultValue('A4')->end()
```

Comment:
```suggestion
                        ->scalarNode('page_size')->defaultValue('A4')->end()
```

Same comment as above